app/controllers/application_controller.rb

```diff
@@ -211,7 +211,8 @@ def set_navigation
                     [
                       ['Events', events_path],
                       # ['Places', places_path],
-                      ['Partners', partners_path]
+                      ['Partners', partners_path],
+                      ['News', news_index_path]
```

Comment:
should only show news link if there is at least one valid news article

Reply:
Can do but that'll be a hit every time someone loads any page on the site

Reply:
maybe one for babys first caching experiment?

Reply:
oh or we could just add a bool to sites thats like "news_enabled?" maybe easier?